app/models/participant_id_change.rb

```diff
@@ -2,8 +2,8 @@
 
 class ParticipantIdChange < ApplicationRecord
   belongs_to :user
-  belongs_to :from_participant, class_name: "User"
-  belongs_to :to_participant, class_name: "User"
+  belongs_to :from_participant, class_name: "User", primary_key: "ecf_id"
```

Comment:
should those be `optional: true`?

Reply:
I guess the question is if I add an id that doesn't exist in the user table will I have an issue here?